platforms/documentation/docs/src/docs/release/notes-template.md

```diff
@@ -31,6 +31,8 @@ Switch your build to use Gradle @version@ by updating the [Wrapper](userguide/gr
 
 `./gradlew wrapper --gradle-version=@version@`
 
+Run the wrapper task again to ensure all files are fully updated.
```

Comment:
💅 Can we change the invocation to `./gradlew wrapper --gradle-version=@version@ && ./gradlew wrapper`, so just copy-pasting it will do the right thing?

Reply:
I like this suggestion!

Reply:
Fixed.